src/Files.App/Actions/Navigation/DuplicateSelectedTabAction.cs

```diff
@@ -31,7 +31,7 @@ public async Task ExecuteAsync(object? parameter = null)
 
 			if (arguments is null)
 			{
-				await NavigationHelpers.AddNewTabByPathAsync(typeof(PaneHolderPage), "Home", true);
```

Comment:
Is it just me or is PaneHolderPage a little more intuitive?

Reply:
I agree with Yair. `MainPanesPage` sounds  like there are non-main panes.

Reply:
What about ShellPanesPage?

I was gonna make them like:
- MainPage
- MainPanePage
- MainShellPage

This order is a-z and also page tree structure(MainPage to MainPanesPage to MainShellPage), so I liked it.

Reply:
> ShellPanesPage

I think it is better than `MainPanesPage`. I have no preference as to whether `ShellPanesPage` or `PaneHolderPage` is better though.

> MainShellPage

Can't we simply say `ShellPage`?

Reply:
I really liked those renaming because of the reasons above. In that case we can remove Main from MainShellPage.

Reply:
@0x5bfa can you change to `PaneHolderPage` or `ShellPanesPage`

Reply:
Done